src/Modal.js

```diff
@@ -22,21 +22,23 @@ class Modal extends React.Component {
     // console.log(this.state.backdropPath)
     return (
       <div className='modal'>
+        <button className='close-modal' onClick={this.handleClick}><img src="../x_icon.png" /></button>
```

Comment:
The pattern of writing out a container function such as handleClick that just calls another function is fine and totally works - but the pattern you'll see more often in the wild is `onClick={() => this.props.toggleModal()}` (where an anonymous function is used to contain the desired function and give the desired function the correct scope). I just wanted to through that out as food for thought. Feel free to dig into it here: https://reactjs.org/docs/handling-events.html